File: C/Grace.c
#include <fcntl.h>
#include <stdio.h>
/*
	This is a comment
*/

#define FILE "Grace_kid.c"
#define CONTENT "#include <fcntl.h>%c#include <stdio.h>%c/*%c%cThis is a comment%c*/%c%c#define FILE %cGrace_kid.c%c%c#define CONTENT %c%s%c%c#define FT()int main() { dprintf(open(FILE, O_WRONLY | O_TRUNC | O_CREAT, 0644), CONTENT, 10, 10, 10, 9, 10, 10, 10, 34, 34, 10, 34, CONTENT, 34, 10, 10, 10); }%c%cFT()"
#define FT()int main() { dprintf(open(FILE, O_WRONLY | O_TRUNC | O_CREAT, 0644), CONTENT, 10, 10, 10, 9, 10, 10, 10, 34, 34, 10, 34, CONTENT, 34, 10, 10, 10); }

FT()

File: Python/Grace.py
FILE = "Grace_kid.py"
CONTENT = "FILE = {2}{0}{2}{1}CONTENT = {2}{3}{2}{1}OPEN = open{1}{1}with OPEN(FILE, {2}w+{2}) as fd:{1}{4}fd.write(CONTENT.format(FILE, chr(10), chr(34), CONTENT, chr(9)))"
OPEN = open

with OPEN(FILE, "w+") as fd:
	fd.write(CONTENT.format(FILE, chr(10), chr(34), CONTENT, chr(9)))